go/pkg/tool/tool.go

```diff
@@ -352,6 +353,40 @@ func (c *Client) DownloadAction(ctx context.Context, actionDigest, outputPath st
 	if _, err := c.GrpcClient.ReadProto(ctx, acDg, actionProto); err != nil {
 		return err
 	}
+
+	if _, err := os.Stat(outputPath); os.IsExist(err) {
```

Comment:
I don't think `Stat` returns "already exists" errors. You probably want `!os.IsNotExist(err)`